freetextresponse/freetextresponse.py

```diff
@@ -297,10 +291,15 @@ def student_view(self, context={}):
                 'other_responses': [],
             }
         )
-        template = get_template('freetextresponse_view.html')
+
+        template = self.loader.render_django_template(
+            'public/templates/freetextresponse_view.html',
+            context=Context(context),
+            i18n_service=self.runtime.service(self, "i18n"),
```

Comment:
Little nitpick, but can you change this to use single quotes? `'i18n'`

Reply:
Done